Update features

katzmo opened the pull request:
Re-export features so they won’t appear already overridden in a fresh installation.

katzmo committed:
Re-export all features → fix markup, dependencies,…

We can’t change the node order in petition templates though, as
nodes need to be in non-alphabetical order for the nodes to get the
expected node ids (petition first, then confirmation page, TYP last).

katzmo committed:
Default templates: fix recipient email addresses

katzmo committed:
Default templates: disable comments in node settings

katzmo commented:
I'd also like to remove `comment` and `diff`variables from the content types (since those modules aren’t actually used and there aren’t any permissions exported either) – any objections?

torotil commented:
> I'd also like to remove `comment` and `diff`variables from the content types (since those modules aren’t actually used and there aren’t any permissions exported either) – any objections?

No.

katzmo committed:
Features: remove comment related variables

katzmo committed:
Features: remove diff module remains

- Leftover dependency
- Deprecated variables

katzmo committed:
Permissions per content type

Move content type specific permissions to the content type feature. This
allows for content types to be disabled/enabled individually.

katzmo committed:
Permissions for contacts to campaignion_supporter

katzmo commented:
Noted a few more issues for later: #81 #82 #83

torotil commented on modules/campaignion_petition/campaignion_petition_templates/campaignion_petition_templates.features.uuid_node.inc:
```diff
@@ -10,56 +10,6 @@
  */
 function campaignion_petition_templates_uuid_features_default_content() {
   $nodes = array();
-
```
This just reverts part of the first commit. Could you squash it into the first commit?

katzmo replied:
Sure. But then it wouldn't point out the underlying issue (#83) and that the order is actually on purpose?

torotil replied:
There is still #83. ;-)

torotil reviewed (approved):
Changes are good to go.

Now we also need them in campaignion_starterkit_uk.

torotil committed:
Fix install order for campaignion_email_to_target_type.

- It’s not required (only some Drupal core modules should use that).
- It should be enabled by default though.

torotil committed:
Add missing dependency on campaignion_action_templates.
